crypto-hash-example/Makefile

```diff
@@ -0,0 +1,7 @@
+run-example: build
+	docker run --rm -ti --name example -v$(PWD):/source example /fluent-bit/bin/fluent-bit -c flb.conf
```

Comment:
This was fail without a TTY (e.g. in CI). Remove the `i`.
It also assumes the current directory is the makefile directory, technically you could run make from another directory and it may fail to find the files.